.gitpod.yml

```diff
@@ -0,0 +1,6 @@
+image:
+  file: .gitpod.Dockerfile
+
+tasks:
+  - init: 'echo "TODO: Replace with init/build command"'
+    command: 'echo "TODO: Replace with command to start project"'
```

Comment:
These lines don't seem to be doing anything, especially when the dev environment doesn't require a server.

```suggestion

```